File: exerciciosemJava/Ex05.java
package exerciciosemJava;

public class Ex05 {
	public static void main(String[] args) {
       
        String string_original = "Douglas Augusto Marcelino Silva";

        char[] string_invertida = new char[string_original.length()];

     
        for (int i = string_original.length()-1, j = 0; i >= 0; i--, j++) {
            string_invertida[j] = string_original.charAt(i);
        }

        String nova_string = new String(string_invertida);

        System.out.println(nova_string);
	}
}

File: Ex05.py

string_original = "Douglas Augusto Marcelino Silva"

string_invertida = []

for i in range(len(string_original)-1, -1, -1):
    string_invertida.append(string_original[i])

nova_string = ''.join(string_invertida)

print(nova_string)
